### DISCUSSION TITLE
Build OpenWrt Firmware Faild: You are running out of disk space.

### DISCUSSION THREAD

**Discussion by AkarinLiu** [actions/virtual-environments#7502] [General questions]
I`m build OpenWrt Firmware, but running faild:
You are running out of disk space. The runner will stop working when the machine runs out of disk space. Free space left: 29 MB
How To Fix this issues?
![image](https://user-images.githubusercontent.com/58081491/234883707-03980ba3-9a4f-4e84-a7bc-cb13511d1838.png)
[logs_6.zip](https://github.com/actions/runner-images/files/11344260/logs_6.zip)
https://github.com/AkarinWRT/openwrt

**Comment by erik-bershel** [comment]
Free disk space does have certain limitations. Because of this, you need to design your actions/jobs in such a way that the process does not run out of available space. 
Unfortunately, at the moment it is difficult to offer you something specific to optimise your process from our side.
